packages/server/dataloader/LocalCache.ts

```diff
@@ -5,6 +5,10 @@ import {RWrite, Updater} from './RethinkDBCache'
 const resolvedPromise = Promise.resolve()
 
 type Thunk = () => void
+
+/**
+ * This cache is only used for the User table, {@see ProxiedCache}, {@see ../db}
```

Comment:
This is great and I'd personally love to see more [JSDocs](https://jsdoc.app/) in our codebase ❤️